src/Stratis.Bitcoin.Features.PoA/Voting/VotingManager.cs

```diff
@@ -272,7 +272,7 @@ public List<IFederationMember> GetFederationFromExecutedPolls()
             }
         }
 
-        public List<IFederationMember> GetModifiedFederation(ChainedHeader chainedHeader)
+        public List<IFederationMember> GetModifiedFederation(ChainedHeader chainedHeader, int offset = 0)
```

Comment:
Please comment the usage and reason for `offset` here.

Reply:
There is a comment:
```
/// <param name="offset">Set to a non-zero value to specify a block relative to <paramref name="chainedHeader"/>.</param>
```